compiler-plugin/src/main/java/io/ballerina/stdlib/log/compiler/staticcodeanalyzer/LogRule.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.org).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.stdlib.log.compiler.staticcodeanalyzer;
+
+import io.ballerina.scan.Rule;
+
+import static io.ballerina.scan.RuleKind.VULNERABILITY;
+import static io.ballerina.stdlib.log.compiler.staticcodeanalyzer.RuleFactory.createRule;
+
+public enum LogRule {
+    AVOID_LOGGING_CONFIGURABLE_VARIABLES(createRule(1, "Avoid logging configurable variables", VULNERABILITY));
```

Comment:
I think we generally describe the rule in terms of what the problem is rather than the fix. E.g., "potentially-sensitive configurable variables are logged".

Reply:
Fixed in [81d0c7d](https://github.com/ballerina-platform/module-ballerina-log/pull/1175/commits/81d0c7d8d9fa4aa8a0814c2882a1a506e15dd47c)